test/api.js

```diff
@@ -18,6 +21,7 @@ test('must be called with new', function (t) {
 generateTests('Without Pool: ', function (options) {
 	options = options || {};
 	options.powerAssert = true;
+	options.pkgDir = options.pkgDir || path.dirname(pkgConf.filepath(conf));
```

Comment:
Could you recompute this on line 11? It's run every time a test needs to create an API which seems unnecessary.